.env

```diff
@@ -23,6 +23,7 @@ _APP_DB_SCHEMA=appwrite
 _APP_DB_USER=user
 _APP_DB_PASS=password
 _APP_DB_ROOT_PASS=rootsecretpassword
+_APP_DB_MAX_CONNECTIONS=251
```

Comment:
We should stop using _APP_DB as a prefix for env vars. We aim to eventually move to `_APP_CONNECTIONS` for all connection related stuff.